client/js/components/GameView/GameOverlays.vue

```diff
@@ -104,6 +113,25 @@ export default {
       }
       return mayCounter;
     },
+    showWaitingForOpponentToDiscardJackFromDeck() {
+      return (
+        this.waitingForOpponentToPlayFromDeck &&
+        this.topTwoCardsCantBePlayed &&
+        (this.opponentPointTotal === 0 || this.opponentQueenCount > 0)
```

Comment:
let's fold this into `topTwoCardsCantBePlayed` since that'd be more semantically accurate. You can't play the top two cards if they are both jacks _and_ jacks can't be played because of a queen or no points to target